modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,141 @@ Exchange Search looks for terms in:
 
-In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+== Searching in Organizations
 
-In Exchange Cloud, you can select which of the organizations visible to you should be included in each search.
+In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+
+In the new Exchange search experience, you can select which of the organizations visible to you should be included in each search. By default, no organization is selected in the drop-down menu, and Exchange searches MuleSoft's public assets and the assets of the master organization and its business groups.
 
 == Save a Search
 
-. Enter a search term and click Save This Search.
-. Exchange prompts you to give the search a name, and to select if the search is for your organization or personal. Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
-. View the left navigation bar to see the saved searches. Exchange lists search terms specific to your business group. Below those terms are saved searches you created.
+The original Exchange search experience and the new Exchange search experience save searches in the same way.
+
+. Enter a search term and click *Save This Search*.
+. Exchange prompts you to give the search a name, and to select if the search is for your organization or personal.
++
+Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
+. View the left navigation bar to see the saved searches.
++
+Exchange lists search terms specific to your business group. Below those terms are saved searches you created.
 +
 image::ex2-saved-searches-groups.png[Saved Search Groups]
 
-== View Assets by Business Group
+== Searching API Specifications
 
-A business group provides a way to group individuals within an organization. A system administrator assigns users to belong to a business group. Each business group can have separate Exchange assets. You need to log into
-Anypoint Platform to view the business groups. Click a group name from the taskbar.
+The original Exchange search experience and the new Exchange search experience process API specifications in the same way.
 
-image::ex2-biz-groups.png[Screenshot - Business groups in upper taskbar drop-down menu]
+Exchange searches these API specification properties:
```

Comment:
c/b "Both searches use the same method to search for these API specification properties:

Reply:
Are the following labels? The capitalization among them is inconsistent.
Are they syntax; should they be formatted as such?